Updated the "getting started" doc section (#1080)
great-expectations/great_expectations: docs/getting_started/typical_workflow.rst

Once the setup is complete, the workflow looks like a loop over the following steps:

1. Data team members capture and document their shared understanding of their data as expectations.
2. As new data arrives in the pipeline, Great Expectations evaluates it against these expectations.
3. If the observed properties of the data are found to be different from the expected ones, the team responds by rejecting (or fixing) the data, updating the expectations, or both.

The article focuses on the "What" and the "Why" of each step in this workflow, and touches on the "How" only briefly. The exact details of configuring and executing these steps are intentionally left out - they can be found in the tutorials and reference linked from each section.

If you have not installed Great Expectations and executed the CLI init command, as described in this :ref:`tutorial<tutorial_init>`, we recommend you do so before reading the rest of the article. This will make a lot of concepts mentioned below more familiar to you.

Setting up a project
----------------------------------------

> To use Great Expectations in a new data project, a :ref:`Data Context<data_context>` must be initialized. You will see references to Data Context throughout the documentation. Data Context simply represents a Great Expectations project.

Reviewer: ```suggestion
To use Great Expectations in a new data project, a :ref:`Data Context<data_context>` should be initialized. You will see references to the Data Context throughout the documentation. A Data Context provides the core services used in a Great Expectations project.
```

When Great Expectations evaluates this expectation against a dataset that has a column named "rating", it returns a validation result saying whether the data meets the expectation.

> This operation can be executed in one of the following compute environments (engines): Pandas, PySpark and an SQL database. This means that the evaluated dataset (or batch of data) can be a Pandas DataFrame, a PySpark DataFrame or a query result set.

Reviewer 2: typo above: `tht` --> `that`

> A :ref:`Data Datasource<datasource>` a connection to a compute environment (a backend such as Pandas, Spark, or a SQL-compatible database) and one or more storage environments.

Reviewer 2: "A Data Datasource a connection" --> "A Datasource is a connection"

> Each You can have multiple Datasources in a project (Data Context). This is useful if the team's pipeline consists of, for example, both a Spark cluster and a Redshift database.

Reviewer 2: ```suggestion
You can have multiple Datasources in a project (Data Context). For example, this is useful if the team’s pipeline consists of both a Spark cluster and a Redshift database.
```

> 

Reviewer 2: below change:

All the Datasources that your project uses are configured in the ``great_expectations/great_expectations.yml`` configuration file of the Data Context:

to

All the Datasources that your project uses are configured in the project's configuration file (``great_expectations/great_expectations.yml``).:

> This operation can be executed in one of the following compute environments (engines): Pandas, PySpark and an SQL database. This means that the evaluated dataset (or batch of data) can be a Pandas DataFrame, a PySpark DataFrame or a query result set.

Reviewer: ```suggestion
```

> A :ref:`Data Datasource<datasource>` is an object Great Expectations uses to connect to a compute environment (e.g., a Postgres database on a particular host).

Reviewer: Using glossary language

```suggestion
A :ref:`Data Datasource<datasource>` a connection to a compute environment (a backend such as Pandas, Spark, or a SQL-compatible database) and one or more storage environments.
```

Multiple sites can be configured inside a project, each suitable for a particular data documentation use case. For example, some data teams use one site that has expectations and validation results from all the runs of their data pipeline for monitoring the pipeline's health, and another site that has only the expectations for communicating with their client (similar to API documentation in software development).

> By default Data Docs sites' files are published to the local filesystem in `great_expectations/uncommitted/data_docs/` directory. To make the site available to the team, a team member can be configure it to publish to a shared location, such as a :ref:`S3<publishing_data_docs_to_s3>` or GCS.

Reviewer: ```suggestion
By default Data Docs sites' files are published to the local filesystem in the `great_expectations/uncommitted/data_docs/` directory. To make a site available more broadly, a team member could configure Great Expectations to publish the site to a shared location, such as a :ref:`S3<publishing_data_docs_to_s3>` or GCS.

> All the Data Docs sites that your project has are defined in the ``great_expectations/great_expectations.yml`` configuration file. The site's configuration defines what they should display and where they are hosted. Data Docs is very customizable, but the details are beyond this article's scope. See this article for details: :ref:`Data Docs Reference<data_docs_reference>`.

Reviewer: ```suggestion
All of your project's Data Docs sites are defined in the ``great_expectations/great_expectations.yml`` configuration file. The site's configuration defines what to compile and where to store results. Data Docs is very customizable;.see the :ref:`Data Docs Reference<data_docs_reference>` for more information.
```

Deploying automated testing into a pipeline
-------------------------------------------

> So far, your team and you used Great Expectations to capture and document your expectations from the data.

Reviewer: This is an abrupt change in voice. I preferred the old tone, but we should settle before making line-in line-out changes

Author: Great - thank you - will update the second half to use the same (more 3rd person/non-prescriptive) voice as the first half.
